sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/src/Generated/Models/SoftDeleteSettings.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.RecoveryServices.Models
+{
+    /// <summary> Soft delete Settings of vault. </summary>
+    public partial class SoftDeleteSettings
```

Comment:
The naming is too general, add it to `prepend-rp-prefix` in [autorest.md](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/src/autorest.md)